charts/kubernetes-external-secrets/README.md

```diff
@@ -6,7 +6,7 @@
 
 ```bash
 $ helm repo add external-secrets https://godaddy.github.io/kubernetes-external-secrets/
-$ helm install external-secrets/kubernetes-external-secrets
+$ helm install --skip-crds external-secrets/kubernetes-external-secrets
```

Comment:
Should it remain:
```bash
$ helm install --name my-release external-secrets/kubernetes-external-secrets
```
?

Which version of Helm are the docs targeting? I cannot think of a way to set the defaults so that the easy install works for both V3 and V2 without one of the versions requiring an additional `--skip-crds` or `--set`.

Reply:
I'd say they've been targeting helm v2 but with EOL of v2 around the corner, I'd say it makes sense to target v3 as a default.